README.md

```diff
@@ -22,9 +22,15 @@ Experience library [documentation](../experience/README.md).
 
 ## Development Notes
 
+### Auth0
+Webextension uses an Auth0 script from <https://github.com/alpheios-project/auth0-chrome>. This repository
+should be installed alongside with `webextension` (both `webextension` and `auth0-chrome` should be located
```

Comment:
perhaps we should make an npm package for our fork of the auth0-chrome library so that it can be a regular dependency? We can switch back to the official distribution if/when they accept our changes.

Reply:
That's a great suggestion, will update the build.

Reply:
We have an alpheios account on npm. If you create an npm user id, I can add you to the organization account so that you can push to it.